charts/victoria-metrics-cluster/templates/vmstorage-statefulset.yaml

```diff
@@ -39,6 +39,12 @@ spec:
       schedulerName: "{{ .Values.vmstorage.schedulerName }}"
 {{- end }}
       automountServiceAccountToken: {{ .Values.serviceAccount.automountToken }}
+      {{- if .Values.vmstorage.initContainers }}
```

Comment:
it can be simplified to 

```yaml
     {{- with .Values.vmstorage.initContainers }}
      initContainers:
        {{- toYaml . | nindent 8 }}
      {{- end }}
```

 also, can you add it to other cluster components and single version?

Reply:
Ok, done